tensorflow/core/kernels/depthwise_conv_grad_op.cc

```diff
@@ -231,7 +231,7 @@ static void CopyOutputBackpropRegion(const DepthwiseArgs& args,
       }
       // Pad to vector-register width (if needed).
       for (int64 d = 0; d < pad_size; ++d) {
-        buffer[buf_base + vectorized_size + scalar_size + d] = 0;
+        buffer[buf_base + vectorized_size + scalar_size + d] = T(0);
```

Comment:
Can we use `static_cast<T>(0)` instead?

Reply:
Thanks @drpngx, done with the change `static_cast<T>(0)` and other similar places.